

try {
    console.log("come√ßando!  \n")
    coisaAleatoria;
} catch(err) {
    console.log('tivemos um erro: \n' + err);
    
}